content/millennium/r4/general-clinical/condition.md

```diff
@@ -73,7 +73,70 @@ _Implementation Notes_
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Update
+
+Update an existing condition
+
+    PUT /Condition/:id
+
+_Implementation Notes_
+
+* Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
```

Comment:
I think we should also mention that currently only problems is supported for update.

Reply:
Added [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/91fee55a512ed5f620c71861b3940e8479c3f61a#diff-c91ea4243fc73689536dc746c972a3d6R84).